lib/mapUtils.js

```diff
@@ -1,74 +1,36 @@
-import BASE from './common';
+import { getAllStores, getProductsByIds } from './airtable/request';
 
-function createStoreData(record) {
-  const data = record.fields;
-  return {
-    name: data['Store Name'],
-    id: data.id,
-    latitude: data.Latitude,
-    longitude: data.Longitude,
-    hours: data['Store Hours'],
-    address: data.Address,
-    products: data.Products,
-    // distance set and used to sort in MapScreen._orderStoresByDistance
-    distance: null,
-    ebt: data['SNAP or EBT Accepted'],
-    rewards: data['Rewards Accepted'],
-  };
+// Adds field 'distance' to stores
+function updateStoreData(record) {
+  return { ...record, distance: null };
 }
 
-function createProductData(record) {
-  const data = record.fields;
-  return {
-    name: data.Name,
-    id: data.id,
-    category: data.Category,
-    points: data.Points,
-    customerCost: data['Customer Cost'],
-    image: data.Image ? data.Image[0].url : null,
-  };
+// Adds field 'imageUrl' to products
+function updateProductData(record) {
+  return { ...record, imageUrl: record.image ? record.image[0].url : null };
 }
 
 // Gets all records in Airtable from the Stores table
 // Returns a promise that resolves to an array of Store objects
-export const getStoreData = function async() {
-  return new Promise((resolve, reject) => {
-    BASE('Stores')
-      .select()
-      .all()
-      .then(records => {
-        const stores = records
-          // Filter out the Clerk Training store
-          .filter(record => record.id !== 'recq6630DixVw63un')
-          .map(record => createStoreData(record));
-        resolve(stores);
-      })
-      .catch(err => reject(err));
-  });
-};
+export async function getStoreData() {
+  const records = await getAllStores();
+  // Filter out the Clerk Training store
+  const stores = records
+    .filter(record => record.id !== 'recq6630DixVw63un')
```

Comment:
Can we (/ should we) also filter out the dev clerk training? `reck65q6wsvyQvnLf`

Reply:
Yup it's probably best to filter both since it's not breaking to do so, and we won't have to update code when we push to PROD